independent-publisher-connectors/Coupa/readme.md

```diff
@@ -0,0 +1,148 @@
+# Coupa
```

Comment:
@NovaGL this is a really great readme file. you can keep the information you have but please try to include all required section from this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md) for raw view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/Independent%20Publisher/readme.md).